docs/extensions/Hyperlink.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
The hyperlink extensions are renamed to HyperlinkExtensions, this doc is not necessary